### PR TITLE
Contrail v4.0 support for Ocata

### PR DESCRIPTION
**Pull request by alexey-mr**

### CODE REVIEW

File: puppet/services/network/contrail-tsn.yaml [hunk 1]
```diff
@@ -21,15 +33,15 @@ parameters:
   NeutronMetadataProxySharedSecret:
     description: Metadata Secret
     type: string
-  VrouterPhysicalInterface:
+  ContrailVrouterPhysicalInterface:
```
Comment (ankitchadha) [inline puppet/services/network/contrail-tsn.yaml]: Modeling is to have Contrail prepended to all the variables that we introduce?

**Comment by alexey-mr** [comment]
@andrey-mp

File: puppet/services/network/contrail-tsn.yaml (hunk 1 above)
Comment (alexey-mr) [inline puppet/services/network/contrail-tsn.yaml]: the base for this pull-request for ocata was taken from the Michael's  review to openstack master, where such prepending done for most variables.

**Comment by Andrey-mp** [comment]
@rrugge

**Comment by Andrey-mp** [comment]
@rrugge

**Pull request merged**